### ISSUE TITLE
Adding an ignore keyword breaks the script 

### ISSUE THREAD

**Issue by skynet01** [alfredapp/google-drive-workflow#22]
### Diagnostic

Thanks for script, decided to add some keyword exclusions and things stopped working... 
If you add a keyword or multiple keywords to ignore list the script doesn't seem to work (always get notification that rebuilding complete) and crashes with 

```
ERROR: Google Drive[Run Script] /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:45:in `block (3 levels) in <main>': undefined method `include?' for #<Pathname:/Volumes/GoogleDrive> (NoMethodError)
```

### Debugger output

```codeblock
[11:56:39.226] Google Drive[Keyword] Passing output '' to Run Script
[11:56:39.408] ERROR: Google Drive[Run Script] /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:45:in `block (3 levels) in <main>': undefined method `include?' for #<Pathname:/Volumes/GoogleDrive> (NoMethodError)
	from /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:45:in `any?'
	from /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:45:in `block (2 levels) in <main>'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/pathname.rb:565:in `block in find'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/find.rb:49:in `block (2 levels) in find'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/find.rb:48:in `catch'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/find.rb:48:in `block in find'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/find.rb:43:in `each'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/find.rb:43:in `find'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/pathname.rb:565:in `find'
	from /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:43:in `each'
	from /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:43:in `each'
	from /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:43:in `block in <main>'
	from /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:42:in `each'
	from /Users/skynet/Library/Application Support/Alfred/Alfred.alfredpreferences/workflows/user.workflow.82F51E7B-A6C9-439A-A91D-55DD8C0032E3/rebuild_cache:42:in `<main>'
[11:56:39.421] Google Drive[Run Script] Processing complete
[11:56:39.422] Google Drive[Run Script] Passing output '' to Run Script
```

**Comment by vitorgalvao** [comment]
Please don’t skip the instructions. I need you to run `!gddiagnostic` and paste the result.

**Comment by skynet01** [comment]
Unfortunately when I run the above nothing gets copied to clipboard

**Comment by vitorgalvao** [comment]
Because it is probably still running. As the instructions say, you have to **wait** for the notification/sound. Also, keep the debugger open to look for possible errors.

Until I have the diagnostic data I cannot properly help you.

**Comment by skynet01** [comment]
I've been using your awesome workflow for month. It's just today I decided to add some exclusions to ignore_list and caching breaks.

Here is the output :
```
### Workflow version
2022.5

### Alfred version
4.6.2

### macOS version
12.1

### Workflow environment variables
Dict {
    ignore_list = picz
    result_limit = 50
    google_drive_path = /Volumes/GoogleDrive/Other computers/
}

### Google drive paths
Exists: /Volumes/GoogleDrive/Other computers/
928M
240605 paths

### Cache dir
cache.db
tmp.db

### Cache file
16777231 8058787 -rw-r--r-- 1 skynet staff 0 42676224 "Jan 19 12:21:04 2022" "Jan 19 12:20:48 2022" "Jan 19 12:20:48 2022" "Jan 19 12:13:43 2022" 4096 84040 0 /Users/skynet/Library/Caches/com.runningwithcrayons.Alfred/Workflow Data/com.alfredapp.googledrive/cache.db

### Temporary cache
16777231 8300411 -rw-r--r-- 1 skynet staff 0 8192 "Jan 19 12:24:21 2022" "Jan 19 12:24:21 2022" "Jan 19 12:24:21 2022" "Jan 19 12:24:21 2022" 4096 16 0 /Users/skynet/Library/Caches/com.runningwithcrayons.Alfred/Workflow Data/com.alfredapp.googledrive/tmp.db

### Build progress
NOT running

### Launchd job
Loaded: -   0   com.alfredapp.launchd.googledrivecache
Installed
```

**Comment by vitorgalvao** [comment]
Thank you. This is fixed in `2022.6`. Also, you can now use wildcards in ignore list.

**Comment by skynet01** [comment]
Love you!
